TACC/TACC-Docs
# fix: exclude internal content pages from search

## Overview

Install and use [`mkdocs-exclude-search`](https://github.com/chrieke/mkdocs-exclude-search) to exclude test pages and includes from search.

## Related

- [Post Launch Docs Meeting](https://docs.google.com/document/d/1cnq4IIb4jI4AFW8o0bPgkBnjRmFrrCkHn8G2NtNzrVM/edit)

## Changes

- **added** mkdocs-exclude-search to dependencies
- **added** mkdocs-exclude-search config to plugins
- **changed** mkdocs config to not search test and include pages

## Testing

1. `docker compose build`
2. `docker compose up`
3. Search for "lmod".
4. Verify results do **not** have:
    > "L" to the "M" to the "O" […]

## UI

| Before | After |
| - | - |
| ![before](https://user-images.githubusercontent.com/62723358/233443945-c4c6b718-363e-4481-bf23-d050c9a4140f.png) | ![after](https://user-images.githubusercontent.com/62723358/233443951-ea672fde-fb98-457e-9643-4ca6bbd186a1.png) |
